### DISCUSSION TITLE
support for android devices & mac

### DISCUSSION THREAD

**Discussion by novawaves** [immense/Remotely#191] [General]
hi is it possible to control android devices and macs? thats very usefull!!

thx

**Comment by stevesobol** [comment]
Mac support is something that's been discussed. I have an Intel-powered Mac mini running Catalina, and would love to see Mac support in Remotely, and I'd like to contribute some code, but honestly, the UI is going to be no big deal - we can just repurpose the Linux UI logic since it uses Avalonia, which also runs on macOS - but writing the code that actually does the screencaps and transmits them to the Remotely server... that's the technical challenge. I would love to take it on when I have time. I just worry that I don't have the technical chops to do it the right way.

Plus, Big Sur changes pretty much everything. I'd need to write one chunk of code to handle older Macs like mine, and another to handle new Macs.

It's something I think I'd enjoy doing... but I can't do it anytime soon.

**Comment by bitbound** [comment]
Like Steve, I also don't have the time to take on a feature of this size right now.  I don't know if I ever will, since my job and family demands aren't going to change in the next few years.  I'd also need to buy a Mac, but the lack of time is the biggest obstacle.
